build-with-stack.dockerfile

```diff
@@ -0,0 +1,75 @@
+# How to use unison in a container?
+# Benefits: No need to install any haskell related dependencies. All you need is podman/docker installed and git installed if you are using the dockerfile source and not from a registry.
+# Container tool used: Podman
+  # Podman is a container tool that has nearly the same CLI interface as Docker with few exceptions. You do not need to run a daemon.
+# 1. Build
+  # Example podman command: 
+  # sudo podman build \
+  # --file build-with-stack.dockerfile \
+  # --build-arg USER_NAME=<1> \
+  # --build-arg USER_EMAIL=<2> \
+  # --build-arg UCM_PORT=<3> \
+  # --build-arg UCM_HOST=<4> \
+  # --build-arg UCM_TOKEN=<5> \
+  # -t unison 
+# 2. Run
+  # Example podman command:
+  # sudo podman run -it --rm unison
+  # The command above will block your terminal. You can either detach Ctrl + P and Ctrl + Q or consult docker documentation or open another terminal and exec into the container to develop *.u files.
+# 3. Develop
+  # Example podman command:
+  # sudo podman exec -it <container name> /bin/bash
+  # After that, you can do your development and UCM will be running. You can attach an additional terminal to see program executions.
+# 4. Interact with source control
+  # You have some options:
+    # You can either copy in your git server ssh key into the container and commit/push from there or you could copy the source out of the container and interact with source control on your host machine. 
+FROM docker.io/archlinux:latest 
```

Comment:
I found some issues using UCM and this linux distro. I will need to fix this. I am probably going to try using a Debian one or better yet the Haskell image from Dockerhub.